### DISCUSSION TITLE
【搜图/视频网站】共5个网站

### DISCUSSION THREAD

**Discussion by wugan-bot156** [share121/inter-knot#1687] [其他]
![无标题5_20230316214410(1)](https://github.com/user-attachments/assets/95488bbc-0ff3-486a-aea2-b0806d50b35b)

# https://yandex.com/images ⬅俄罗斯的网站可搜p站、X等
# https://saucenao.com ⬅p站和X
# https://www.google.com/imghp ⬅p站、X等
# https://www.tineye.com ⬅以图搜图
# https://trace.moe ⬅以图搜番

**Comment by lllliufeng** [comment]
感谢share.

**Comment by capacitor1** [comment]
补充：

## https://ascii2d.net/

搜图网站

**Comment by Myitian** [comment]
还有 IQDB
[https://iqdb.org/](https://iqdb.org/)

**Comment by Myitian** [comment]
![图片](https://github.com/user-attachments/assets/fcd0ccae-b9a7-4fa4-b826-eddc23362c3a)
可以使用这个插件：**Image Search Options**，由 SauceNao 作者开发
右键图片即可选择引擎搜索

[下载页面地址
![图片](https://github.com/user-attachments/assets/65845b3d-180a-4c90-9d12-16dbb9590331)](https://saucenao.com/tools/)
页面上的四个链接是 Chrome/Firefox/旧版Firefox/crx直下
- Chrome 版支持的搜索引擎少
- Firefox 版好几年没更新了，有些搜索引擎用不了
